src/NuGet.Core/NuGet.Commands/RestoreCommand/LockFileBuilder.cs

```diff
@@ -16,6 +16,11 @@
 
 namespace NuGet.Commands
 {
+    internal class MaccatalystFallback
+    {
+        internal bool _usedXamarinIOs = false;
```

Comment:
Would a property be better? Is there a perf. concern?

Reply:
what would using a property gain me?

Reply:
Better code readability? Sometimes, when working with VS, the editor flags underscore variables with naming violations. I think it's fine for now.